### PR TITLE
Update ukrainian.yml

### PR DESCRIPTION
**Pull request by n3k0q**

### CODE REVIEW

File: SCPDiscordPlugin/Languages/ukrainian.yml [hunk 1]
```diff
@@ -20,28 +20,28 @@ global_regex: [
   #"SCPs":"",
   "FoundationForces":"МОГ",
   "ChaosInsurgency":"Повстанець хаосу",
-  #"Scientists":"",
+  "Scientists":"Вчений",
```
Comment (n3k0q) [inline SCPDiscordPlugin/Languages/ukrainian.yml]: change scientists to Науковець please

Comment (KarlOfDuty) [reply SCPDiscordPlugin/Languages/ukrainian.yml]: But this is your pull request? You can change whatever you want?

Comment (KarlOfDuty) [reply SCPDiscordPlugin/Languages/ukrainian.yml]: And you have not allowed me to edit your branch so I can't edit it for you either.

File: SCPDiscordPlugin/Languages/ukrainian.yml [hunk 2]
```diff
@@ -20,28 +20,28 @@ global_regex: [
...
+  "Scientist":"Науковець",
   "Scp079":"SCP-079",
   "ChaosConscript":"Новобранець Хаосу",
   "Scp096":"SCP-096",
   "SCP-0492":"SCP-049-2",
   "NtfSergeant":"Сержант МОГ",
   "NtfCaptain":"Капітан МОГ",
   "NtfPrivate":"Рядовий МОГ",
-  #"Tutorial":"",
+  "Tutorial":"Навчання",
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/ukrainian.yml:44]: The ones commented out here shouldn't be used, they are just there for reference. If you look higher up you can see there are already translations for these phrases.

Comment (n3k0q) [reply SCPDiscordPlugin/Languages/ukrainian.yml:44]: oh, sorry

**Comment by n3k0q** [comment]
i guess, translation is done, if u want u can accept 😎

**Comment by KarlOfDuty** [comment]
Sure, but do you have time to add the new language entries from this patch?
image

You can find them in the english file.

**Comment by n3k0q** [comment]
oh, ill do it

**Comment by KarlOfDuty** [comment]
Thanks!

**Comment by n3k0q** [comment]
what means "ranked player"?

**Comment by KarlOfDuty** [comment]
It means players with a rank like admin, moderator

**Comment by n3k0q** [comment]
a, ok lemme make another commit

**Comment by n3k0q** [comment]
i think it's all?

**Comment by n3k0q** [comment]
oh i forgor add botactivity
lemme make it

**Comment by KarlOfDuty** [comment]
Looks good, thanks! Is there anything you would like to change or is it ready to merge?

**Review by KarlOfDuty — APPROVED** [review]

**Comment by n3k0q** [comment]
i guess it's ready to merge

**Comment by n3k0q** [comment]
a
wait

**Comment by n3k0q** [comment]
lemme do smth

**Comment by n3k0q** [comment]
now its ready to merge

**Comment by KarlOfDuty** [comment]
:+1:

**Pull request merged**